website/docs/index.html.markdown

```diff
@@ -373,6 +373,76 @@ The following arguments are used to configure the VMware Cloud Director Provider
 * `import_separator` - (Optional; *v2.5+*) The string to be used as separator with `terraform import`. By default
   it is a dot (`.`).
 
+* `ignore_metadata` - (Optional; *v3.10+*) Use one or more of these blocks to ignore specific metadata entries.
```

Comment:
I feel we're missing something like "... from being managed by this Terraform provider".

When you first see this feature without context the above is not instantly clear.

Same ask for the variable's description in the schema.

Reply:
Done